src/lib.rs

```diff
@@ -453,7 +453,8 @@ static mut GLOBAL_STATE: GlobalState = GlobalState {
 /// use dcv_color_primitives as dcp;
 /// dcp::initialize();
 /// ```
-pub fn initialize() {
+pub fn initialize(#[cfg(feature = "test_instruction_sets")] force_set: &str) {
```

Comment:
This is a bit weird...

Shall we have a `intialize_with_instructionset` variant instead?

Reply:
Yes. Seems more clean.